packages/components/data-table-manager/_docs/usage-example.js

```diff
@@ -12,10 +14,24 @@ const columns = [
   { key: 'country', label: 'Country' },
 ];
 
-const Example = () => (
+export const Example = () => (
   <DataTableManager columns={columns}>
     <DataTable rows={rows} />
   </DataTableManager>
 );
-
 export default Example;
+
+// Introduce this component to test that DataTable and DataTableManager should not necessarily be direct descendants
+const SomeOtherComponent = () => {
+  return <div>Some other component</div>;
+};
+
+// With the data table settings decoupled.
+//  It can also be exported as default.
+export const ExampleWithDecoupledDataTableManager = () => (
+  <DataTableManagerProvider>
+    <DataTableManager columns={columns} />
+    <SomeOtherComponent />
+    <DataTable rows={rows} />
+  </DataTableManagerProvider>
```

Comment:
I wonder if this will work? Shouldn't columns be passed to the provider?

Reply:
Yes, that is correct.
updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/37d6c7c20176cd563a6ad743408f2df3a22a752f

Thank you